src/test/shell/testenv.sh.tmpl

```diff
@@ -323,6 +323,7 @@ EOF
         "rules_license"
         "rules_proto"
         "rules_python"
+        "rules_pkg"
```

Comment:
I think you'll have to mark rules_pkg as `test_WORKSPACE_files` in distdir_deps.bzl to make this actually work, otherwise it won't fail but fall back to download it.

Reply:
Done